de.marw.cdt.cmake.core/src/main/java/de/marw/cdt/cmake/core/internal/BuildscriptGenerator.java

```diff
@@ -147,13 +155,66 @@ private IFolder getBuildFolder() {
       String buildPathString = buildP.toString();
       buildPathString = mngr.resolveValue(buildPathString, "", "",
           ManagedBuildManager.getDescriptionForConfiguration(config));
-      buildP= new Path(buildPathString);
+      buildP = new Path(buildPathString);
     } catch (CdtVariableException e) {
       log.log(new Status(IStatus.ERROR, CdtPlugin.PLUGIN_ID,
           "variable expansion for build directory failed", e));
     }
 
-    buildFolder = project.getFolder(buildP);
+    // if the path is absolute, it is necessary to link the external directory
+    // into the project to use it
+    if (buildP.isAbsolute()) {
+      buildFolder = project.getFolder("build_output");
```

Comment:
This name should be configurable for the end user. May some special input String in the project config page (e.g. /build_output->/abs/filesystem/path?)